operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,198 @@
+
+See https://github.com/Cray-HPE/docs-csm/blob/CASMPET-6934-SBPS-Node-Personalization/operations/configuration_management/iSCSI_SBPS_Node_Customization.md for more info.
+
+## Steps to configure and run GOSS tests 
+In order to verify the readiness of the iSCSI targets before triggering the boot of compute nodes/ UAN's, we need to run GOSS tests as sanity checks on iSCSI targets.
+
+See https://github.com/Cray-HPE/sbps-marshal/blob/CASMPET-7214-goss-sbps-doc/GOSS_tests_for_sbps.md for more info.
+
+## IMS images tagging
+To trigger the boot of compute node/ UANs, the images (rootfs/ PE ) are tagged to determine which rootfs image to be projected for compute/ UANs as there will be many rootfs/ PE images. SBPS Marshal agent uses key/value pair of sbps-project/true to identify the image(s) tagged. IMS images (rootfs) tagging is supported from IMS initially which requires to be done manually using craycli. 
+
+Command to tag an IMS image using craycli 
+```bash
+cray ims images update < Image id > --metadata-operation set --metadata-key < key > --metadata-value < value >
+```
+e.g.
+```bash
+cray ims images update bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb --metadata-operation set --metadata-key sbps-project --metadata-value true
+```
+Command to describe the IMS image metadata:
+```bash
+cray ims images describe < Image id > --format json
+e.g
+
+# cray ims images describe f1d6c8fe-32e2-420a-a051-377e34a8bd8a --format json
+{
+  "arch": "x86_64",
+  "created": "2024-07-24T20:59:37.469889",
+  "id": "f1d6c8fe-32e2-420a-a051-377e34a8bd8a",
+  "link": {
+    "etag": "af7e30ed3219ee4e5c227c5f734dca2d",
+    "path": "s3://boot-images/f1d6c8fe-32e2-420a-a051-377e34a8bd8a/manifest.json",
+    "type": "s3"
+  },
+  "metadata": {},
+  "name": "compute-uss-1.1.0-134-csm-1.5.x86_64"
+}
+
+Or
+
+cray ims images list --format=json | jq '.[] | select(.id == <Image id>)'
+e.g.
+
+# cray ims images list --format=json | jq '.[] | select(.id == "bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb")'                                    
+ {
+  "arch": "x86_64",
+  "created": "2024-07-18T22:05:16.565885",
+  "id": "bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb",
+  "link": {
+    "etag": "3325f830ba9ec291005a4087be4f666f",
+    "path": "s3://boot-images/bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb/manifest.json",
+    "type": "s3"
+  },
+  "metadata": {
+    "name": "secure-storage-ceph-6.1.94-x86_64.squashfs",
+    "sbps-project": "true"   <---------------- Tagged with key/value pair sbps-project/true 
+  },
+  "name": "secure-storage-ceph-6.1.94-x86_64.squashfs"
+}
+```
+
+Command to untag an IMS image using craycli
+```bash
+cray ims images update < Image id > --metadata-operation remove --metadata-key < key >

```

Comment:
Again, remove the `bash` so the `<>`s don't get displayed in red, and:

- `< Image id >` -> `<image_id>`
- `< key >` -> `<key>`